src/utils/deep_merge.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import objectAssign from "./object_assign";
+
+/**
+ * Check if an item is an object
+ * @param item
+ * @returns {boolean}
+ */
+function isObject(item: unknown) : boolean {
+  return (item !== null
+          && item !== undefined
```

Comment:
Redundant with typeof item === "object", no? Or is there some perf implication, I'm not seeing?

Reply:
Still the case?

Reply:
yes, will be changed